ush/gaussian_sfcanl.sh

```diff
@@ -133,63 +132,77 @@ export PGM=$GAUSFCANLEXE
 export pgm=$PGM
 $LOGSCRIPT
 
-iy=${PDY:0:4}
-im=${PDY:4:2}
-id=${PDY:6:2}
-ih=${cyc}
+gcycle_dates=("${PDY}${cyc}")
+if [[ "${DOIAU:-}" == "YES" ]]; then  # Create Gaussian sfcanl file at beginning of window
+  half_window=$(( assim_freq / 2 ))
+  BDATE=$(date --utc -d "${PDY} ${cyc} - ${half_window} hours" +%Y%m%d%H)
+  gcycle_dates+=("${BDATE}")
+fi
 
 export OMP_NUM_THREADS=${OMP_NUM_THREADS_SFC:-1}
 
-# input interpolation weights
-${NLN} "${FIXWGTS}" "./weights.nc"
+for ii in "${!gcycle_dates[@]}"; do
+
+  gcycle_date=${gcycle_dates[ii]}
+
+  if [[ ! -d "${DATA}/gausfcanl_${gcycle_date:8:2}" ]]; then
+    mkdir -p "${DATA}/gausfcanl_${gcycle_date:8:2}" 
+  fi
+
+  cd "${DATA}/gausfcanl_${gcycle_date:8:2}"
```

Comment:
```suggestion
  rm -rf "${DATA}/gausfcanl_${gcycle_date:8:2}"
  mkdir -p "${DATA}/gausfcanl_${gcycle_date:8:2}" 
  cd "${DATA}/gausfcanl_${gcycle_date:8:2}"
```
We should perhaps start with a clean directory each time, so that when/if the job fails and restarts, it gets a clean directory.